lib/router/mocks/mediatorMock.js

```diff
@@ -0,0 +1,41 @@
+var sinon = require('sinon');
+require('sinon-as-promised');
+var sampleUser = require('../fixtures/user');
+
+
+exports.request = function(topic, params) {
+  var stub = sinon.stub();
+  // valid credentials stub
+  stub.withArgs('wfm:user:auth', {username: sampleUser.username, password: sampleUser.password}).resolves(true);
+
+  // invalid username stub
+  stub.withArgs('wfm:user:auth',
+    sinon.match({
+      username: sinon.match(function(username) {
+        return username !== sampleUser.username;
+      }),
+      password: sinon.match(sampleUser.password)
+    })).rejects(new Error());
```

Comment:
Small thing, could probably add messages for the errors and assert the relevant message.

Otherwise we don't know if the test failed because of an error from the user or an error from the password.